boards/deprecated.cmake

```diff
@@ -434,7 +434,9 @@ set(mps3_an547_DEPRECATED
 set(mps3_an547_ns_DEPRECATED
     mps3/an547/ns
 )
-
+set(mps3_an547_tsv_DEPRECATED
+    mps3/an547/tsv
+)
```

Comment:
this is only needed for boards that are converted from HWMv1 to v2.

`mps3_an547_tsv` doesn't exist as a hwmv1 board.
```suggestion
```